package-deps-hash/src/getPackageDeps.ts

```diff
@@ -36,7 +34,7 @@ export function getPackageDeps(packagePath: string = process.cwd(), excludedPath
 
           processOutputBlocks[1]
             .trim()
-            .split(os.EOL)
+            .split('\n')
```

Comment:
You should add a comment explaining that this is intentional over os.EOL and also provide context why